### PR TITLE
Add badges to favicon

### LINKED ISSUE #728: Favicon counter
It would be nice to have a favicon counter in Uptime Kuma. It displays how many monitors are up or down while browsing in other tabs.

**Example:**
image

**Uptime Robot**
image

### PR DESCRIPTION
**Pull request by willianrod**
# Description

Resolves #728

- Added favicon badges counter 

## Type of change
- New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and test it
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)
image
image

### CODE REVIEW

**Comment by deefdragon** [comment]
It looks good so far, but I am gonna ask that you add an option to enable/disable this.
Also, looking at the code, it looks like it always displays a number, even if there are 0 down. I think it should probably remove the badge if there are no down monitors.

**Comment by WillianRod** [comment]
> It looks good so far, but I am gonna ask that you add an option to enable/disable this.
> 
> 
> 
> Also, looking at the code real quick, it looks like it always displays a number, even if there are 0 down. I think it should probably remove the badge if there are no down monitors.

The Favicon library automatically hide the badge when 0 is passed.

An option to hide it could be interesting, but this badge is a more visual way to call attention when a monitor goes down.

File: src/pages/StatusPage.vue [hunk 1]
```diff
@@ -209,12 +209,18 @@ import ImageCropUpload from "vue-image-crop-upload";
...
+import Favico from "favico.js";
+
 const toast = useToast();
 
 const leavePageMsg = "Do you really want to leave? you have unsaved changes!";
 
 let feedInterval;
 
+const favicon = new Favico({
+    animation: "none"
```
Comment (Saibamen) [inline src/pages/StatusPage.vue:221]:
```suggestion
    animation: "none",
```

**Review by Saibamen — APPROVED** [review]

**Comment by deefdragon** [comment]
> > It looks good so far, but I am gonna ask that you add an option to enable/disable this.
> > Also, looking at the code real quick, it looks like it always displays a number, even if there are 0 down. I think it should probably remove the badge if there are no down monitors.
> 
> The Favicon library automatically hide the badge when 0 is passed.

Understood.
 
> An option to hide it could be interesting, but this badge is a more visual way to call attention when a monitor goes down.

So, my brain is annoyingly reliant on something's general color and shape for recognition (especially in icons like favicons/profile pics), and adding this badge _may_ make it more difficult for me figure out which of my billion tabs has UK on it when I get a ping if I want more details. Its possible that this won't actually impact things, but I would rather have the option than not.

Its minor, and I can definitely see the trade-off of settings clutter vs configuration, but I just want to explain why I want the option in a bit more detail.

**Comment by louislam** [comment]
Just tested, it is working.

However, I think using vue's `watch` should be better and easier. I will come back later.

**Comment by R0GGER** [comment]
This feature would be very nice to have... Any progress over here?

**Pull request merged**